projects/1.18/assets/nuclearscience/nuclearscience/lang/zh_cn.json

```
@@ -0,0 +1,132 @@
+{
+  "item.nuclearscience.uranium235" : "铀-235",
+  "item.nuclearscience.uranium238" : "铀-238",
+  "item.nuclearscience.lifthf4uf4" : "LiF-ThF4-UF4 盐混合物",
+  "item.nuclearscience.flinak" : "FLiNaK 盐混合物",
+  "item.nuclearscience.yellowcake" : "黄饼",
+  "item.nuclearscience.fissiledust" : "裂变物质粉",
+  "item.nuclearscience.plutoniumoxide" : "四氧化钚",
+  "item.nuclearscience.plutonium239" : "钚-239",
+  "item.nuclearscience.polonium210" : "钋-210",
+  "item.nuclearscience.polonium210chunk" : "钋-210 碎块",
+  "item.nuclearscience.thorianitedust" : "方钍石粉",
+  "item.nuclearscience.cellempty" : "空单元",
+  "item.nuclearscience.celldeuterium" : "氘单元",
+  "item.nuclearscience.cellheavywater" : "重水单元",
+  "item.nuclearscience.celltritium" : "氚单元",
+  "item.nuclearscience.cellelectromagnetic" : "电磁单元",
+  "item.nuclearscience.cellantimattersmall" : "反物质单元 500 mg",
+  "item.nuclearscience.cellantimatterlarge" : "反物质单元 4 g",
+  "item.nuclearscience.cellantimatterverylarge" : "反物质单元 12 g",
+  "item.nuclearscience.celldarkmatter" : "暗物质单元",
+  "item.nuclearscience.fuelheuo2" : "高浓缩二氧化铀燃料",
+  "item.nuclearscience.fuelleuo2" : "低浓缩二氧化铀燃料",
+  "item.nuclearscience.fuelspent" : "用尽的燃料棒",
+  "item.nuclearscience.fuelplutonium" : "钚燃料棒",
+  "item.nuclearscience.geigercounter" : "盖革计数器",
+  "item.nuclearscience.hazmatboots" : "防化靴子",
+  "item.nuclearscience.hazmathelmet" : "防化头盔",
+  "item.nuclearscience.hazmatlegs" : "防化护腿",
+  "item.nuclearscience.hazmatplate" : "防化胸甲",
+  "item.nuclearscience.antidote" : "解毒剂",
+  "item.nuclearscience.canisterlead" : "衬铅储罐",
+  "item.nuclearscience.frequencycard" : "射频识别卡",
+  "block.nuclearscience.gascentrifuge" : "气体离心机",
+  "block.nuclearscience.nuclearboiler" : "核能锅炉",
+  "block.nuclearscience.chemicalextractor" : "化学提取机",
+  "block.nuclearscience.radioisotopegenerator" : "放射性同位素温差发电机",
+  "block.nuclearscience.freezeplug" : "防冻塞",
+  "block.nuclearscience.turbine" : "蒸汽涡轮机",
+  "block.nuclearscience.reactorcore" : "裂变反应堆芯",
+  "block.nuclearscience.electromagnet" : "电磁铁",
+  "block.nuclearscience.electromagneticglass" : "电磁玻璃",
+  "block.nuclearscience.electromagneticbooster" : "电磁推进器",
+  "block.nuclearscience.electromagneticswitch" : "电磁开关",
+  "block.nuclearscience.fusionreactorcore" : "聚变反应堆芯",
+  "block.nuclearscience.fuelreprocessor" : "燃料再处理机",
+  "block.nuclearscience.radioactiveprocessor" : "放射性处理机",
+  "block.nuclearscience.msrfuelpreprocessor" : "MSR 燃料预处理机",
+  "block.nuclearscience.plasma" : "等离子",
+  "block.nuclearscience.particleinjector" : "粒子注入器",
+  "block.nuclearscience.quantumcapacitor" : "量子电容",
+  "block.nuclearscience.teleporter" : "传送机",
+  "block.nuclearscience.controlrodassembly" : "控制棒组件",
+  "block.nuclearscience.blocklead" : "陶瓷-铅辐射屏蔽器",
+  "block.nuclearscience.msrreactorcore" : "MSR 反应堆芯",
+  "block.nuclearscience.heatexchanger" : "热交换机",
```

```suggestion
  "block.nuclearscience.heatexchanger" : "热交换器",
```
[术语在线](https://www.termonline.cn/search?k=Steam%20Turbine&r=1645465289053)